### PR TITLE
Added e2e test for avocode email tags input

### PR DESCRIPTION
**Pull request by kiranparajuli589**
Test for avocode email tags input (Basic and Unique) has been added.

### CODE REVIEW

File: packages/avocode-email-tagsinput/test/e2e/basic.spec.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    for (const testEmail of unsupportedEmailEdgeCases) {
+      it(`should not be allowed to add ${testEmail} as an email tag`, async () => {
+        await addEmailTag(page, testEmail)
+        await assertEmailTagCount(page, 0)
+      })
+    }
+
+    for (const testEmail of supportedEmailEdgeCases) {
+      it(`should not be allowed to add ${testEmail} as an email tag`, async () => {
```
[the remark this answers is not in the document]
Comment (kiranparajuli589) [reply packages/avocode-email-tagsinput/test/e2e/basic.spec.js]: done!

**Comment by phil-davis** [comment]
Sometimes Travis fails, e.g. 
https://travis-ci.org/github/JankariTech/avocode-email-tagsinput/builds/675157225

https://travis-ci.org/github/JankariTech/avocode-email-tagsinput/builds/675164395

Is there some timing problem or?

Please have a look. We could run CI a few times in Travis to see if it is an intermittent problem.

**Comment by Talank** [comment]
> Sometimes Travis fails, e.g.
> https://travis-ci.org/github/JankariTech/avocode-email-tagsinput/builds/675157225
> 
> https://travis-ci.org/github/JankariTech/avocode-email-tagsinput/builds/675164395
> 
> Is there some timing problem or?
> 
> Please have a look. We could run CI a few times in Travis to see if it is an intermittent problem.

I am not quite sure what the problem is.  In the build https://travis-ci.org/github/JankariTech/avocode-email-tagsinput/builds/675157225 a scenario `CollapsibleTagsinput › counter › multiple tags › should update count when tags are added and input is on multiple lines` fails, and in the build https://travis-ci.org/github/JankariTech/avocode-email-tagsinput/builds/675175832 the failing scenario is `TagsInput › selection › should select the whole text using mouse` .This PR does not alter any of those scenarios. However, all the tests pass when I run it locally.

**Comment by phil-davis** [comment]
Maybe the current tests are already not so stable. Have a look in the Travis of the upstream repo and see if they get "random" fails.

**Comment by Talank** [comment]
many of the [PR-build](https://travis-ci.org/github/avocode/avocode-email-tagsinput/pull_requests) have failed, but not in [build history](https://travis-ci.org/github/JankariTech/avocode-email-tagsinput/builds)

File: packages/avocode-email-tagsinput/test/e2e/basic.spec.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+    for (const testEmail of supportedEmailEdgeCases) {
+      it(`should be allowed to add ${testEmail} as an email tag`, async () => {
+        await addEmailTag(page, testEmail)
+        await assertEmailTagCount(page, 1)
+      })
+    }
+
+    it('should be possible to add email tags with full numeric local-part', async () => {
+      const testEmail = '1234@test.co'
```
Comment (individual-it) [inline packages/avocode-email-tagsinput/test/e2e/basic.spec.js:97]: why is that not in the table supportedEmailEdgeCases ?

File: packages/avocode-email-tagsinput/test/e2e/basic.spec.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      await page.click(tagsInputFieldSelector)
+      await page.type(tagsInputFieldSelector, 'abcd')
+      await page.keyboard.press('ArrowLeft')
+      const [ anchorOffset ] = await getAnchorAndFocus(page)
+      expect(anchorOffset).to.equal(3)
+    })
+  })
+
+  describe('user typing different email edge cases', () => {
+    const unsupportedEmailEdgeCases = [
```
Comment (individual-it) [inline packages/avocode-email-tagsinput/test/e2e/basic.spec.js:71]: what about having just "normal" not email address input? where are we testing those?

**Comment by individual-it** [comment]
tests fail in the PR build and seem not to run in the push build

File: packages/avocode-email-tagsinput/test/e2e/basic.spec.js (hunk 3 above)
Comment (kiranparajuli589) [reply packages/avocode-email-tagsinput/test/e2e/basic.spec.js:71]: added inside unsupportedEmailEdgeCases array!

File: packages/avocode-email-tagsinput/test/e2e/basic.spec.js (hunk 2 above)
Comment (kiranparajuli589) [reply packages/avocode-email-tagsinput/test/e2e/basic.spec.js]: fixed!

**Comment by Talank** [comment]
the changes in this PR have been merged to `upstream master` and `master` of this forked repo has been rebased with the upstream master. So, I am closing this PR
